README.md

```diff
@@ -1,86 +1,254 @@
 
-Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
 
-<sub>Remember to star it, flow will love you more :)</sub>
+Dedicated to make your workflow flow more seamlessly. Search everything from applications, files, bookmarks, YouTube, Twitter and more. Flow will continue to evolve, designed to be open and built with the community at heart.
+
+[<img width="12px" src="https://user-images.githubusercontent.com/26427004/104119722-9033c600-5385-11eb-9d57-4c376862fd36.png"> **SOFTPEDIA EDITOR'S PICK**](https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml)
+
+## 🎉 New Features in 1.9 
+
+- All New Design. New Themes, New Setting Window. Animation & Sound Effect, Color Scheme aka Dark Mode.
+- New Plugins, Plugin Store, Game Mode, Wizard window
+
+<a href="https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.9.0">Full changelog</a>
 
 ---
 
 <h4 align="center">
-  <a href="#Features">Features</a> •
-  <a href="#Getting-Started">Getting Started</a> •
+  <a href="#Getting-Started">Getting Started</a> •  <a href="#Features">Features</a> • <a href="#Plugins">Plugins</a> •
+       <a href="#Hotkeys">Hotkeys</a> •
   <a href="#QuestionsSuggestions">Questions/Suggestions</a> •
   <a href="#Development">Development</a> •
-  <a href="https://flow-launcher.github.io/docs">Documentation</a>
+  <a href="https://flow-launcher.github.io/docs">Docs</a>
 </h4>
 
+
 ---
 
+## 🚗 Getting Started
+
+### Installation
+
+| [Windows 7+ installer](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Setup.exe) | [Portable](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Portable.zip) | `WinGet install "Flow Launcher"` |
+| :----------------------------------------------------------: | :----------------------------------------------------------: | :------------------------------: |
+
+> Windows may complain about security due to code not being signed, this will be completed at a later stage. If you downloaded from this repo, you are good to continue the set up. 
+
+And You can download <a href="https://github.com/Flow-Launcher/Flow.Launcher/discussions" target="_blank">early access version</a>.
+
+
 ## Features
 
-![The Flow](https://user-images.githubusercontent.com/26427004/82151677-fa9c7100-989f-11ea-9143-81de60aaf07d.gif)
+### Applications & Files
+
+<img src="https://user-images.githubusercontent.com/6903107/144517107-efd83586-9084-46e0-b7dc-79a6748ea936.png" width="500">
 
-- Search everything from applications, files, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.
+- Flow searches files and contents.
 - Search for file contents.
```

Comment:
```suggestion
- Search for files or their contents.
```

Reply:
think we only need this line `Search for files or their contents.`

Reply:
applied.